### PR TITLE
Initial commit of code

### PR DESCRIPTION
**Pull request by yahehe**

### CODE REVIEW

File: rakefile.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
+require 'bundler/setup'
```
Comment (jnallard) [inline rakefile.rb:1]: We don't need the rakefile anymore, right?

Comment (yahehe) [reply rakefile.rb:1]: If Appveyor doesn't need it then no

Comment (yahehe) [reply rakefile.rb:1]: We can make other changes outside of this PR, this is just to get the code in

File: src/Logging/MancyAppender.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
+﻿using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using log4net.Appender;
+using log4net.Core;
+
+namespace Cimpress.Nancy.Logging
+{
+    public class MancyAppender : AppenderSkeleton
```
Comment (jnallard) [inline src/Logging/MancyAppender.cs:9]: You have several classes still named MancySomething. Do we want to rename those?

Comment (yahehe) [reply src/Logging/MancyAppender.cs:9]: Yep

File: src/project.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Cimpress.Nancy",
+  "title": "Cimpress.Nancy",
+  "version": "1.0.0-*",
+  "description": "A helper library for handling reproducible tasks such as logging and authentication for Nancy microservices",
+  "copyright": "Cimpress 2017",
+  "authors": [ "cdowney", "joallard", "NancyFX Guild" ],
```
Comment (jnallard) [inline src/project.json:7]: Maybe you should change these to our Github usernames?

**Pull request merged**